include/ceed/jit-source/gallery/ceed-mass2dbuild.h

```diff
@@ -19,10 +19,11 @@ CEED_QFUNCTION(Mass2DBuild)(void *ctx, const CeedInt Q,
   // *INDENT-OFF*
   // in[0] is Jacobians with shape [2, nc=2, Q]
   // in[1] is quadrature weights, size (Q)
-  const CeedScalar (*J)[2][CEED_Q_VLA] = (const CeedScalar(*)[2][CEED_Q_VLA])in[0],
-                                    *w = in[1];
+  typedef CeedScalar array_t[3][CEED_Q_VLA];
```

Comment:
I don't think that we want to force users to have to do this.

Reply:
For sure. This is a temporary hack since there is a deadline next week.

I will update the OCCA parser to handle pointers-to-arrays correctly in the near future.